Samples/SampleInput/MeadowApp.cs

```diff
@@ -60,7 +60,7 @@ void InitializeGlade()
         {
             LogService.Log.Trace("Starting Glade2d...");
             glade = new Game();
-            glade.Initialize(display, 2, EngineMode.RenderOnDemand, RotationType._90Degrees);
+            glade.Initialize(display, 1, EngineMode.RenderOnDemand, RotationType._90Degrees);
```

Comment:
Changing the display scaling in the demo significantly reduces perf. I don't care about this too much but should the demo app have better perf and larger graphics by default so new devs have a better experience on the small PL board screen?

Reply:
Resolved via slack convo. We'll leave this like this for now as we work through perf issues.